scripts/hid_configurator/configurator_cli.py

```diff
@@ -45,7 +45,7 @@ def perform_dfu(dev, args):
         return
     img_ver_dev = info.get_fw_version()
 
-    img_file = DfuImage(args.dfu_image, info, dev.get_board_name())
+    img_file = DfuImage(args.dfu_image, info, dev)
```

Comment:
Maybe we could only pass board name and bootloader name here? To avoid passing data that should not be used by the DfuImage class.